admin/scripts/modules/infopult.xtpl

```diff
@@ -32,9 +32,9 @@
         Před odkliknutím zkontroluj že proběhlo:
         <ul>
           <li>Předání trička, placky, stravenek, číslo pokoje, srovnání nedoplatku</li>
-          <li>Předání manuálu účastníka a vysvětlení last moment přihlašování.</li>
+          <li>Vysvětlení last moment přihlašování.</li>
           <li>Doplnění chybějících údajů, jako adresy, telefonu</li>
-          <li>Vyplnění čísla OP do sdíleného dokumentu</li>
```

Comment:
Dochází ke změně významu věty. Point na kontrolu celého OP není od věci, ale nevím jak probíhá konkrétně soupiska údajů pro město Pardubice. Jestli to opravdu není manuálně zapisování do dokumentu, který se odevzdává. Je to minimálně na konzultaci s tím kdo má na starosti infopult.

Reply:
Úprava vychází z loňského roku na infopultu, kdy se číslo OP vyplňovalo pouze do Osobních údajů jako Čislo dokladu, zatímco tento text byl naposledy dle git blame upravován 2022.